docs/variable-group.md

```diff
@@ -0,0 +1,121 @@
+# Variable Group
+
+Create
+[variable group](https://docs.microsoft.com/en-us/azure/devops/pipelines/library/variable-groups)
+in Azure DevOps project.
+
+Usage:
+
+```
+spk service [command] [options]
```

Comment:
@sarath-p you mean `spk variable-group`?

Reply:
yes, thanks for catching copy/paste error!